src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/resource/CalculationController.kt

```diff
@@ -477,6 +475,32 @@ class CalculationController(
     return offenderKeyDatesService.getNomisCalculationSummary(offenderSentCalculationId)
   }
 
+  @GetMapping(value = ["/nomis-calculation-summary/booking/{bookingId}/calculation/{offenderSentCalculationId}"])
```

Comment:
is the plan that this endpoint is used by the UI?
reason I'm asking is that I thought this was going to be a solely back end thing ; i.e. wehenever they get the  key dates info (e.g. latest calc) we lookuo to see if it was overridden and modify the hint text accordingly